src/styles/code.scss

```diff
@@ -7,6 +7,7 @@ code:not([class]) {
   border-radius: var(--amplify-radii-small);
   display: inline-block;
   line-height: var(--amplify-line-heights-small);
+  font-style: normal;
```

Comment:
Sometimes there are `<code>` elements within a blockquote, this is so the monospace font doesn't get italicized.